config/recipe-hyprland.yml

```diff
@@ -28,6 +28,37 @@ modules:
       - hyprland
       - waybar
       - xdg-desktop-portal-hyprland
+      - xorg-x11-server-Xwayland
+
+      # wayland utilities
+      - swaybg
+      - swayidle
+      - swaylock
+      - wlr-randr
+      - wl-sunset
+
+      # essentials for usability
+      - pipewire
+      - wireplumber
```

Comment:
there are many duplicates here that are already included in https://github.com/wayblueorg/wayblue/blob/live/config/common-packages.yml

Please remove those from this list